### PR TITLE
Increase coverage to more than 50%

### PR DESCRIPTION
**Pull request by kayoub5**

### CODE REVIEW

File: Test/DeviceFixture.cs [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+SharpPcap is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with SharpPcap.  If not, see <http://www.gnu.org/licenses/>.
+*/
+/*
+ * Copyright 2010 Chris Morgan <chmorgan@gmail.com>
```
Comment (chmorgan) [inline Test/DeviceFixture.cs:18]: you should put your name here I think

File: Test/DeviceFixture.cs [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        public DeviceFixture(ICaptureDevice device)
+        {
+            Device = device;
+        }
+
+        public ICaptureDevice GetDevice() => Device;
+
+        public override string ToString()
+        {
+            return new string(Device.Name.Take(22).ToArray());
```
Comment (chmorgan) [inline Test/DeviceFixture.cs:48]: why 22?

Comment (kayoub5) [reply Test/DeviceFixture.cs:48]: I don't need the full device name, and if `ToString()` returns a very long string, Visual Studio test explorer will look weird.
with 22 chars Visual Studio Test Explorer looks like this
image

Comment (chmorgan) [reply Test/DeviceFixture.cs:48]: @kayoub5 imo we shouldn't use magic numbers, could you modify the code to be something like:

var maximumDeviceNameCharactersToDisplay = 22;
blah.Take(maximumDeviceNameCharactersToDisplay);

That way we can tell from the code at least what you were intending to do.

Comment (chmorgan) [reply Test/DeviceFixture.cs:48]: @kayoub5 can you make that code change here to clarify that magic number? See my above comment.

File: Test/LivePcapDeviceSetFilterTest.cs [hunk 3]
```diff
@@ -10,45 +11,39 @@ namespace Test
...
-            catch (DeviceNotReadyException)
-            {
-                caughtExpectedException = true;
-            }
-
-            Assert.IsTrue(caughtExpectedException, "Did not catch the expected PcapDeviceNotReadyException");
+            var device = fixture.GetDevice();
+            Assert.Throws<DeviceNotReadyException>(
+                () => device.Filter = "tcp port 80",
+                "Did not catch the expected PcapDeviceNotReadyException"
```
Comment (chmorgan) [inline Test/LivePcapDeviceSetFilterTest.cs:45]: it looks like the exception name changed but the text didn't with some prior changes. Can you update the text here to say DeviceNotReadyException and not PcapDeviceNotReadyException?

File: Test/DeviceFixture.cs (hunk 2 above)
Comment (kayoub5) [reply Test/DeviceFixture.cs]: Updated to take full `Device.Name`

**Pull request merged**